concourse/tasks/deploy-to-govuk-paas.yml

```diff
@@ -0,0 +1,35 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: governmentpaas/cf-cli
+    tag: e1ffec0d1940706f157a8c1e0ab8131b7084fa1c
+inputs:
+  - name: git-main
+    path: src
+params:
+  CF_API: https://api.london.cloud.service.gov.uk
+  CF_USERNAME: ((paas-username))
+  CF_PASSWORD: ((paas-password))
+
+run:
+  dir: src
+  path: sh
+  args:
+    - "-c"
+    - |
+      set -eu
+
+      cf api "$CF_API"
+      cf auth
+      cf t -o "$CF_ORG" -s "$CF_SPACE"
+      cf create-app $CF_APP_NAME || true
+      cf apply-manifest -f manifest.yml
+      cf set-env $CF_APP_NAME CF_STARTUP_TIMEOUT "$CF_STARTUP_TIMEOUT"
+
+      cf scale -i "$APP_INSTANCES" $CF_APP_NAME
+
+      cf set-env $CF_APP_NAME REDIRECT_BASE_URL "https://www.${CDN_DOMAIN}"
+
+      cf push $CF_APP_NAME --strategy rolling
+      cf map-route $CF_APP_NAME "$CDN_DOMAIN" --hostname www
```

Comment:
Does this need to be anything more than just

```
cf api "$CF_API"
cf auth
cf t -o "$CF_ORG" -s "$CF_SPACE"
cf push
```

?

Reply:
I have simplified this here: [concourse/tasks/deploy-to-govuk-paas.yml](https://github.com/alphagov/govuk-account-static-errors/commit/7f5dcb987e9ecfe5d796d71f7bfa9b0873f53b63)

Reply:
I still like keeping the deploy items in a separate file. It matches my expectations from other GOV.UK PaaS apps and I find it easer to read through the file with the separation.